### PR TITLE
Rspec

### PR DESCRIPTION
**Pull request by franciscoabalan**
## Summary
Installation and configuration rspec unit test to models. Add database cleaner, faker and factory bot gems.

## Trellocard
https://trello.com/c/qxOnE396/20-crear-tests-para-el-modelo-de-book

## Screenshots
Captura de Pantalla 2019-10-07 a la(s) 11 26 23

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,6 +41,12 @@ group :development, :test do
   gem 'capybara', '>= 2.15'
   gem 'selenium-webdriver'
   gem 'rubocop', '~> 0.59.2', require: false
+  #TEST
+  gem 'rspec-rails', '~> 3.8'
```
Comment (aleochoam) [inline Gemfile:47]: As you said, this gems are only  for the testing environment, you can move it  to the :test group

File: spec/factories.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'faker'
+
+FactoryBot.define do
```
Comment (aleochoam) [inline spec/factories.rb]: We usually have one factory per file, called user_factory.rb, book_factory.rb

**Review by aleochoam — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (franciscoabalan) [reply Gemfile:47]: Done

File: spec/factories.rb (hunk 2 above)
Comment (franciscoabalan) [reply spec/factories.rb]: Done

**Review by aleochoam — APPROVED** [review]
👏

**Review by joredudiaz — APPROVED** [review]

**Pull request merged**